cmd/xl-storage-disk-id-check.go

```diff
@@ -579,3 +508,314 @@ func (p *xlStorageDiskIDCheck) updateStorageMetrics(s storageMetric, paths ...st
 		}
 	}
 }
+
+const (
+	diskHealthOK = iota
+	diskHealthOffline
+)
+
+// diskMaxConcurrent is the maximum number of running concurrent operations
+// for local and (incoming) remote disk ops respectively.
+var diskMaxConcurrent = 50
+
+func init() {
+	if s, ok := os.LookupEnv("_MINIO_DISK_MAX_CONCURRENT"); ok && s != "" {
```

Comment:
can we move this to let's say a new sub-system called `drive` and 

```
mc admin config set alias/ drive max-io=100
```

my plan is also to move the `disable-odirect` from API here once we bring it in here..

Reply:
I don't really want this to be user tweakable.

Reply:
why not @klauspost we should have such provisions IMO since we do not know which ones work well for NVMe v/s HDDs yet..

Reply:
Actually this is not needed, looks like 50 per drive is enough.